regions/core/core.py

```diff
@@ -249,13 +249,16 @@ def mpl_properties_default(self, shape='patch'):
 
         return kwargs
 
-    def plot(self, ax=None, **kwargs):
+    def plot(self, origin=(0, 0), ax=None, **kwargs):
         """
         Calls as_patch method forwarding all kwargs and adds patch
         to given axis.
 
         Parameters
         ----------
+        origin : array_like, optional
+            The ``(x, y)`` position of the origin of the displayed image.
```

Comment:
could you clarify: these are x,y positions in pixel units, right?  Just to make sure they're not confused with data units (e.g., WCS)

Reply:
yes, pixel units. Perhaps should we accept `PixCoord` instead of a tuple to avoid confusion?

Reply:
That's... a good idea from a programmer's perspective, as it makes the code more robust, but it's really inconvenient for something as simple as a pixel coordinate, since it requires an additional import and a much longer function call.  I am weakly in favor of accepting a tuple and just documenting it.